evennia/utils/validfuncs.py

```diff
@@ -0,0 +1,202 @@
+"""
```

Comment:
For consistency with other such modules, please rename to `validatorfuncs.py`. This makes it a little clearer what they do at a glance.